### PR TITLE
Qradar create query and hunting

### PR DESCRIPTION
**Pull request by David-BMS**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/42493

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@michalgold Doc review completed.

File: Packs/QRadar/ReleaseNotes/2_2_1.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#### Playbooks
+##### New: QRadar Get Hunting Results
+- This playbook is used to to sort the QRadar search results to display the IP addresses, assets and usernames that the search provided. In addition results allow to to differentiate between internal and external IP addresses as well as query the QRadar assets API in order to get the assets details from the IP addresses. The user can provide the QRadar fields names and the organizations IP ranges in order to properly sort the data. The end result of the playbook will be the internal and external IP addresses detected as well as the assets and users. (Available from Cortex XSOAR 6.0.0).
```
Comment (idovandijk) [inline Packs/QRadar/ReleaseNotes/2_2_1.md:3]: fix "to to" (mistake appears twice)

**Review by idovandijk — CHANGES_REQUESTED** [review]
* `QRadar Build Query and Search` - this playbook seems good :)
* `QRadar Get Hunting Results` - Fix logical issue, also uses "qradar-get-assets" which is deprecated
* Update playbook image + readme and fix
* Fix RN comment
* Fix fromversion to 6.2

**Review by idovandijk — APPROVED** [review]
Add documentation to `base_field_match` to say that it makes one of the optional fields mandatory, other than this - good to go :)

**Pull request merged**